news/fixTestNews.rst

```diff
@@ -0,0 +1,18 @@
+**Added:** None
+
+**Changed:**
+
+* test/test_news.py
+  It now uses regex to verify the format of rst files
+
+**Deprecated:** None
+
+**Removed:** None
+
+**Fixed:**
+
+* Previous implementation had a lot of nested if-elses
+  and was needlessly tedious
```

Comment:
BTW, the news files are all merged together when a release happens.  This comment doesn't make a lot of sense out of context, so I will trim it down.